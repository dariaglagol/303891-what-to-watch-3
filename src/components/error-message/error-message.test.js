import React from "react";
import renderer from "react-test-renderer";
import ErrorMessage from "./error-message";

const mockedResponse = {
  error: `error`
};

it(`error render`, () => {
  const errorMessage = renderer
    .create(
        <ErrorMessage
          response={mockedResponse}
        />
    ).toJSON();

  expect(errorMessage).toMatchSnapshot();
});
