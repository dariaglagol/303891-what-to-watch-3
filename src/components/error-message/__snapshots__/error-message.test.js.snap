// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`error render 1`] = `
<div
  className="error-message"
>
  <h1
    className="user-page__title"
  >
    error
  </h1>
</div>
`;
